src/lib/gl_format_color.js

```diff
@@ -9,24 +9,22 @@
 
 'use strict';
 
-var tinycolor = require('tinycolor2');
 var isNumeric = require('fast-isnumeric');
+var rgba = require('color-rgba');
```

Comment:
Can you add `color-rgba` to the package.json and push again?

I'll be interesting to see if the tests pass on CI ...